### PR TITLE
Initial plugin folder setup

### PR DESCRIPTION
**Pull request by geoffreydhuyvetters**
### Added

- initial setup for plugin architecture
- CRA in js folder with topbar menu

### Changed

- moved .prettierrc to the root of the project

### CODE REVIEW

File: hat/assets/js/test/dom.js [hunk 1]
```diff
@@ -1,39 +0,0 @@
-import { JSDOM } from 'jsdom';
```
Comment (geoffreydhuyvetters) [inline hat/assets/js/test/dom.js:1]: @kaiobrito Can you provide a bit of context here, we did we delete these?

File: hat/templates/iaso/index.html [hunk 2]
```diff
@@ -1,29 +1,18 @@
-{% extends 'iaso/base.html' %}
-{% load i18n %}
-{% load render_bundle f
-{% block extrajs %}
-    <script type="text/javascript">
-        var STATIC_URL = "{{ STATIC_URL }}";
-    </script>
-{% endblock %}rom webpack_loader %}
```
Comment (geoffreydhuyvetters) [inline hat/templates/iaso/index.html:8]: Fixed a malformed piece of the template here.

File: hat/assets/js/test/dom.js (hunk 1 above)
Comment (kaiobrito) [reply hat/assets/js/test/dom.js:1]: It was my mistake, this folder shouldn't be deleted

Comment (geoffreydhuyvetters) [reply hat/assets/js/test/dom.js]: Moved it back, no worries

**Review by xavierbarbosa — APPROVED** [review]
Looks good! 👍

**Review by kaiobrito — APPROVED** [review]

**Pull request merged**